app/viewmodels/nominees/NomineeIndividualSummaryHelper.scala

```diff
@@ -0,0 +1,129 @@
+      messagePrefix = "individualNomineesPhoneNumber.mainPhoneNumber")
+
+  def nomineeAltPhone: Option[SummaryListRow] =
+    answerAlternativePhoneNo(IndividualNomineesPhoneNumberPage,
+      routes.IndividualNomineesPhoneNumberController.onPageLoad(CheckMode),
+      messagePrefix = "individualNomineesPhoneNumber.alternativePhoneNumber")
+
+  def nomineeHasNino: Option[SummaryListRow] =
+    answerPrefix(IsIndividualNomineeNinoPage,
+      routes.IsIndividualNomineeNinoController.onPageLoad(CheckMode),
+      messagePrefix = "isIndividualNomineeNino")
+
+  def nomineeNino: Option[SummaryListRow] =
+    answerPrefix(IndividualNomineesNinoPage,
+      routes.IndividualNomineesNinoController.onPageLoad(CheckMode),
+      messagePrefix = "individualNomineesNino")
+
+  def nomineePassportNumber: Option[SummaryListRow] = None // TODO individual passport page
+
+  def nomineePassportCountry: Option[SummaryListRow] = None // TODO individual passport page
+
+  def nomineePassportExpiry: Option[SummaryListRow] = None // TODO individual passport page
+
+  def nomineeAddress: Option[SummaryListRow] =
+    answerAddress(NomineeIndividualAddressLookupPage,
+      controllers.addressLookup.routes.NomineeIndividualAddressLookupController.initializeJourney(CheckMode),
+      messagePrefix = "nomineeIndividualAddress")
+
+  def nomineeAddressChanged: Option[SummaryListRow] =
+    answerPrefix(IsIndividualNomineePreviousAddressPage,
+      routes.IsIndividualNomineePreviousAddressController.onPageLoad(CheckMode),
+      messagePrefix = "isIndividualNomineePreviousAddress")
+
+  def nomineePreviousAddress: Option[SummaryListRow] = None // TODO individual nominee prev address
+
+  def nomineeCanBePaid: Option[SummaryListRow] =
+    answerPrefix(IsIndividualNomineePaymentsPage,
+    routes.IsIndividualNomineePaymentsController.onPageLoad(CheckMode),
+    messagePrefix = "isIndividualNomineePayments")
+
+  def nomineeAccountName: Option[SummaryListRow] =
+    answerAccountName(IndividualNomineesBankDetailsPage,
+    routes.IndividualNomineesBankDetailsController.onPageLoad(CheckMode),
+    messagePrefix = "individualNomineesBankDetails.accountName")
+
+  def nomineeSortCode: Option[SummaryListRow] =
+    answerSortCode(IndividualNomineesBankDetailsPage,
+      routes.IndividualNomineesBankDetailsController.onPageLoad(CheckMode),
+      messagePrefix = "individualNomineesBankDetails.sortCode")
+
+  def nomineeAccountNumber: Option[SummaryListRow] =
+    answerAccountNumber(IndividualNomineesBankDetailsPage,
+      routes.IndividualNomineesBankDetailsController.onPageLoad(CheckMode),
+      messagePrefix = "individualNomineesBankDetails.accountNumber")
+
+  def nomineeBuildingRoll: Option[SummaryListRow] =
+    answerRollNumber(IndividualNomineesBankDetailsPage,
+      routes.IndividualNomineesBankDetailsController.onPageLoad(CheckMode),
+      messagePrefix = "individualNomineesBankDetails.rollNumber")
+
+  val rows: Seq[Seq[SummaryListRow]] = Seq(
```

Comment:
its not readable, can we make it more struchured like 
NomineeSummary(nomineeType: Seq[SummaryListRow], details: Seq[SummaryListRow],
  bankDetails: Seq[SummaryListRow], individualDetails: Seq[SummaryListRow])
This may common to both Indvidula and Oranisation, we can cansider empty Seq for Optional values

Reply:
on it